syncer/syncer.go

```diff
@@ -1195,6 +1226,30 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 		cancel()
 	}()
 
+	// start background task to get/update current ts offset between dm and upstream
```

Comment:
Should we start syncing after we get at least one `tsOffset` successfully?

Reply:
added